cmplibrary/src/test/java/com/sourcepoint/cmplibrary/SourcePointClientTest.java

```diff
@@ -33,33 +33,31 @@
 @Config(manifest = Config.NONE, sdk = O)
 public class SourcePointClientTest {
 
-    AsyncHttpClient http;
-    SourcePointClient sourcePointClient;
-    ConsentLib.OnLoadComplete onLoadComplete;
+    private AsyncHttpClient http;
+    private SourcePointClient sourcePointClient;
+    private ConsentLib.OnLoadComplete onLoadComplete;
 
     @Before
     public void setSourcePointClient() throws ConsentLibException.BuildException {
         http = mock(AsyncHttpClient.class);
         onLoadComplete = mock(ConsentLib.OnLoadComplete.class);
 
-        sourcePointClient = new SourcePointClientBuilder(808, "TEST", true).build();
+        sourcePointClient = new SourcePointClientBuilder(123, "example.com", true).build();
         sourcePointClient.setHttpDummy(http);
-        System.out.println("TTTT : 11 " + http);
     }
 
     JSONObject jsonObject = null;
 
-    private void doAPICallWithAnswer(final boolean onSuccess) {
+    private void doAPICallWithAnswer(final boolean onSuccess , String response) {
         doAnswer(new Answer() {
             @Override
             public Object answer(InvocationOnMock invocation) {
-                System.out.println("TTTT : CALLED");
                 SourcePointClient.ResponseHandler listener = (SourcePointClient.ResponseHandler) invocation.getArguments()[1];
 
                 if (onSuccess) {
                     String stringToParse = "{\"site_id\":\"http://google.com\"}";
```

Comment:
Since now we're using the `response` parameter, do we still need this line?
```java
String stringToParse = "{\"site_id\":\"http://google.com\"}";
```

Reply:
I will remove this string, its not required anymore

Reply:
removed the unused string variable